addon/-private/system/relationships/ext.js

```diff
@@ -12,8 +12,8 @@ var Map = Ember.Map;
 var MapWithDefault = Ember.MapWithDefault;
 
 var relationshipsDescriptor = Ember.computed(function() {
-  if (Ember.testing === true && relationshipsDescriptor._cacheable === true) {
-    relationshipsDescriptor._cacheable = false;
+  if (Ember.testing === true && relationshipsDescriptor._volatile === true) {
```

Comment:
should'nt the condition be inversed ? Per @krisselden comment: "Should not use relatedTypesDescriptor._cacheable = false; in testing as this is now _volatile = true"